xml/operations-maintenance-controller-onetwo_controller_recovery.xml

```diff
@@ -111,25 +111,30 @@
      </para>
     </note>
     <para>
-     Before proceeding, you may want to take a look at
-     <emphasis role="bold">info/server_info.yml</emphasis> to see if the
-     assignment of the node you have added is what you expect. It may not be,
-     as nodes will not be numbered consecutively if any have previously been
-     removed. This is to prevent loss of data; the config processor retains
-     data about removed nodes and keeps their ID numbers from being
-     reallocated. See the Persisted Server Allocations section in for
-     information on how this works.
+     Before proceeding, look at <emphasis
+     role="bold">info/server_info.yml</emphasis> to see if the assignment of
+     the node you have added is what you expect. It may not be, as nodes will
+     not be numbered consecutively if any have previously been removed. This is
+     to prevent loss of data; the configuration processor retains data about
```

Comment:
"To prevent loss of data, the configuration processor retains data about removed nodes and keeps their ID numbers from being reallocated."